package.json

```
@@ -8,8 +8,11 @@
     "deploy": "sls deploy"
   },
   "dependencies": {
+    "@types/node": "^16.11.7",
```

```suggestion
    "@types/node": "^16.11.7",
```
해당 내용은 의존성보다는 개발 의존성에 위치해도 무관한 파일입니다.

타입의 경우 개발할때는 필요하지만, 빌드시에는 오히려 포함되면 용량만 잡아먹기때문입니다.